src/javascript/app/pages/user/new_account/welcome_page.js

```diff
@@ -1,20 +1,22 @@
-const BinarySocket   = require('../../../base/socket');
-const Client         = require('../../../base/client');
-const localize       = require('../../../../_common/localize').localize;
-const createElement  = require('../../../../_common/utility').createElement;
-const getElementById = require('../../../../_common/common_functions').getElementById;
-const Url            = require('../../../../_common/url');
+const BinarySocket      = require('../../../base/socket');
+const Client            = require('../../../base/client');
+const localize          = require('../../../../_common/localize').localize;
+const createElement     = require('../../../../_common/utility').createElement;
+const getElementById    = require('../../../../_common/common_functions').getElementById;
+const Url               = require('../../../../_common/url');
+const showLoadingImage  = require('../../../../_common/utility').showLoadingImage;
 
 const WelcomePage = (() => {
     const onLoad = () => {
         BinarySocket.wait('authorize', 'landing_company', 'get_settings').then(() => {
+            const welcome_msg = getElementById('welcome_container');
```

Comment:
@brandon-binary becuase it's a javascript element, seems better to use el_welcome_container or el_welcome_message instead